src/libstd/time.rs

```diff
@@ -59,6 +59,21 @@ pub use core::time::Duration;
 ///    println!("{}", now.elapsed().as_secs());
 /// }
 /// ```
+///
+/// # Underlying System calls
+/// Currently, the following system calls are being used to get the current time using `now()`:
+///
+/// |  Platform |               System call                        |
+/// |:---------:|:------------------------------------------------:|
+/// | Cloud ABI | [clock_time_get (Monotonic Clock)](https://github.com/NuxiNL/cloudabi/blob/master/cloudabi.txt) |
+/// |    SGX    | Not implemented                                  |
```

Comment:
Is this true? I don't know anything about this target but looking at the libstd/sys/sgx/time.rs and going from there, it seems to be implemented.

Reply:
Yeah I took a look at that code as well. And when I further inspected the code and followed the references:

It first calls (in sys/sgx/abi/usercalls/mod.rs):
```
/// Usercall `insecure_time`. See the ABI documentation for more information.
#[unstable(feature = "sgx_platform", issue = "56975")]
pub fn insecure_time() -> Duration {
    let t = unsafe { raw::insecure_time() };
    Duration::new(t / 1_000_000_000, (t % 1_000_000_000) as _)
}
```
which calls [this](https://docs.rs/fortanix-sgx-abi/0.3.3/fortanix_sgx_abi/struct.Usercalls.html#method.insecure_time)
If you look at the source code this is unimplemented(). I do not really understand why though...

Reply:
The start of the page https://docs.rs/fortanix-sgx-abi/0.3.3/fortanix_sgx_abi/struct.Usercalls.html says 

> This is not a real structure, it's just a convenient way to group functions with rustdoc.

so I assume the code behind the `[src]` link there isn't the real code. Notice how all the other methods on that struct (corresponding to many other usercalls) are also all `unimplemented!()`.

cc @jethrogb to be sure

Reply:
Thanks for quick response. To be totally clear though: we found that documentation but were unsure whether / where the "usercall" is actually implemented -- that is, if you run a Rust program using `Instant::now` or `SystemTime::now` in an enclave, (how) will those calls work? I guess whether it will work is a bit of a silly question since docs clearly imply it will, but the source link pointing at an `unimplemented!()` was a bit confusing.

Reply:
Yeah we don't need more implementation details. Just wanted to be 100% sure that "SGX | `insecure_time` usercall" is the correct row to add to this table. Thanks!